src/theme/separator.ts

```diff
@@ -0,0 +1,116 @@
+export default (config: { colors: string[] }) => ({
+  slots: {
+    root: 'flex items-center align-center text-center',
+    border: '',
+    container: 'font-medium text-gray-700 dark:text-gray-200 flex',
+    icon: 'shrink-0 size-5',
+    avatar: 'shrink-0',
```

Comment:
```suggestion
    avatar: 'shrink-0 size-5',
```

Reply:
I'm considering adding `size-5` everywhere there is an avatar defined (which matches the `2xs`), this would allow customization in the same place as icon. What do you think?

Reply:
Forcing `size-5` will require you to both edit this and Avatar's `size` prop, example:
```diff
<USeparator
  :avatar="{
    src: 'https://avatars.githubusercontent.com/u/739984?v=4',
+    size: 'lg'
  }"
+  :ui="{ avatar: 'size-9' }"
  decorative
  orientation="vertical"
/>
```
I'm more down to not set a specified size for it, or remove the hardcoded `size="2xs"`

Reply:
also, if we do specify `size-5` then we should also force the `text-[10px]` then.

IMO this is a skip